From jaswinder@kernel.org  Mon Aug 17 12:40:56 2009
From: Jaswinder Singh Rajput <jaswinder@kernel.org>
Date: Fri, 14 Aug 2009 11:54:37 +0530
Subject: Staging: otus: includecheck fix: drivers/staging/otus/usbdrv.h
To: Greg KH <greg@kroah.com>, devel@driverdev.osuosl.org, Greg Kroah-Hartman <gregkh@suse.de>
Message-ID: <1250231077.2912.2.camel@ht.satnam>



fix the following 'make includecheck' warning:

  drivers/staging/otus/usbdrv.h: linux/usb.h is included more than once.

Signed-off-by: Jaswinder Singh Rajput <jaswinderrajput@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/otus/usbdrv.h |    4 ----
 1 file changed, 4 deletions(-)

--- a/drivers/staging/otus/usbdrv.h
+++ b/drivers/staging/otus/usbdrv.h
@@ -61,10 +61,6 @@
 #define USB_REG_IN_PIPE                     3
 #define USB_REG_OUT_PIPE                    4
 
-#if (WLAN_HOSTIF == WLAN_USB)
-#include <linux/usb.h>
-#endif
-
 #ifdef ZM_HOSTAPD_SUPPORT
 #include "athr_common.h"
 #endif
